File: allomeufs-community-moderation/src/main/java/snb/projects/domain/mappers/UsersMappers.java
package snb.projects.domain.mappers;

import snb.projects.domain.models.users.User;
import snb.projects.domain.models.users.UserCreated;
import org.mapstruct.Mapper;
import org.mapstruct.MappingConstants;

@Mapper(componentModel = MappingConstants.ComponentModel.CDI)
public interface UsersMappers {
	UserCreated fromUsersToUsersLoggedIn(User user, String jwToken);
}

File: allomeufs-community-moderation/src/main/kotlin/snb/projects/domain/mappers/UsersMappers.kt
package snb.projects.domain.mappers

import snb.projects.domain.models.users.User
import snb.projects.domain.models.users.UserCreated
import org.mapstruct.Mapper
import org.mapstruct.MappingConstants

@Mapper(componentModel = MappingConstants.ComponentModel.JAKARTA)
fun interface UsersMappers {
    fun fromUsersToUsersLoggedIn(user: User, jwToken: String): UserCreated
}